imports/ui/helpers/time.coffee

```diff
@@ -0,0 +1,44 @@
+Session.setDefault('now', +new Date)
+
+tickInterval = 1000
+
+tick = ->
+  Session.set('now', Session.get('now') + tickInterval)
+setInterval tick, tickInterval
+
+talkTime = do ->
+  second = 'second'
```

Comment:
No, I'm talking about replacing the second/minute/hour etc. variables with the strings "second"/"minute"/"hour"